Formula/x/x265.rb

```diff
@@ -22,6 +23,7 @@ class X265 < Formula
   end
 
   def install
+    ENV.runtime_cpu_detection
```

Comment:
Haven't checked if this is actually accurate, but the build fails without it, so 🤞

Seems like it should be though.